core/config/chaintype.go

```diff
@@ -10,20 +10,21 @@ type ChainType string
 
 // nolint
 const (
-	ChainArbitrum ChainType = "arbitrum"
-	ChainMetis    ChainType = "metis"
-	ChainOptimism ChainType = "optimism"
-	ChainXDai     ChainType = "xdai"
+	ChainArbitrum 		 ChainType = "arbitrum"
+	ChainMetis    		 ChainType = "metis"
+	ChainOptimism 		 ChainType = "optimism"
+	ChainOptimismBedrock ChainType = "optimismBedrock"
+	ChainXDai     		 ChainType = "xdai"
 )
 
 var ErrInvalidChainType = fmt.Errorf("must be one of %s or omitted", strings.Join([]string{
-	string(ChainArbitrum), string(ChainMetis), string(ChainOptimism), string(ChainXDai),
+	string(ChainArbitrum), string(ChainMetis), string(ChainOptimism), string(ChainXDai), string(ChainOptimismBedrock),
 }, ", "))
 
 // IsValid returns true if the ChainType value is known or empty.
 func (c ChainType) IsValid() bool {
 	switch c {
-	case "", ChainArbitrum, ChainMetis, ChainOptimism, ChainXDai:
+	case "", ChainArbitrum, ChainMetis, ChainOptimism, ChainOptimismBedrock, ChainXDai:
```

Comment:
We have the linter set up to require adding this to all switches, like below in `IsL2()`. (it just doesn't flag code outside the PR...seems flawed in this situation :thinking: )

Reply:
Where is that configuration? Default case covers the rest and we have a default case in every switch statement that uses ChainType, so it might be that.

Reply:
The linter will normally still insist that the cases are 'exhaustive' https://golangci-lint.run/usage/linters/#exhaustive
IIUC, it is just ignoring the other switches since this PR didn't modify that code directly